Tasks/SonarQubePostTest/PRCA/Orchestrator.ps1

```diff
@@ -14,7 +17,7 @@ function HandleCodeAnalysisReporting
         InitPostCommentsModule $vssConnection
         $comments = GetCommentsFromIssues $newIssues
         
-        PostAndResolveComments $comments "SonarQube Code Analysis"
+        PostAndResolveComments $comments "SonarQube Code Analysis" "This code analysis issue was resolved by recent changes"
```

Comment:
When a comment is resolved, we push a "resolved" message to the associated thread. This differentiates message that the user manually resolved from those that PRCA resolves.Personally I don't like this as it takes too much space, but I'll run this by Jean-Marc and Blair.